cli/copr_cli/build_config.py

```diff
@@ -41,7 +41,7 @@
 {%- endif %}
 {%- if repos %}
 
-config_opts[config_opts['package_manager'] + '.conf'] += \"\"\"
+config_opts['dnf.conf'] += \"\"\"
```

Comment:
Maybe a dumb question, but we are still releasing `copr-cli` for EPEL7. Won't this cause problems?

Reply:
The dnf.conf is equivalent to config option yum.conf, so we should be fine even on el7, even building for el7.